packages/appstore/src/modules/tour-guide/tour-guide.tsx

```diff
@@ -67,39 +56,39 @@ const TourGuide = () => {
         </div>
     );
 
-    if (joyride_index === 0) {
-        tour_step_locale.next = (
-            <div
-                onClick={() => {
-                    trackStepForward(7);
-                }}
-            >
-                <Localize i18n_default_text='Next' />
-            </div>
-        );
+    tour_step_locale.next = (
+        <div
+            style={{ padding: '0.9rem' }}
```

Comment:
we are removing padding in the styles config and then adding it back here. I truly it's not a matter of a css issue but it is an implementation issue.